user-stories.md

```diff
@@ -0,0 +1,8 @@
+# Пользовательские истории:
+- Как пользователь, я хочу зарегистрироваться и получить свой аккаунт с уникальным id, чтобы меня можно было отличить от других пользователей.
+- Как пользователь, я хочу получать короткую уникальную ссылку по длинному URL, чтобы было проще её использовать.
+- Как пользователь, я хочу использовать выданную мне короткую ссылку и переходить на соответствующий сайт без задержек.
+- Как зарегистрированный пользователь, я хочу иметь возможность просматривать уже созданные мною ранее сокращения для ссылок, чтобы использовать их повторно.
```

Comment:
We have two different interpretations of this story with Dmitri.
Interpretations:

1. User can see a list of all of her shot links, in order to be able to remember them.
2. User can assign new link to her old shot links

So, what does it mean?